net/host.go

```diff
@@ -123,30 +129,15 @@ func NewHost(cfg *Config) (*host, error) {
 
 	// set libp2p host options
 	opts := []libp2p.Option{
-		libp2p.ListenAddrs(addr),
-		libp2p.DisableRelay(),
-		libp2p.Identity(key),
+		libp2p.ListenAddrStrings(
+			fmt.Sprintf("/ip4/%s/tcp/%d", listenIP, cfg.Port),
+			fmt.Sprintf("/ip4/%s/udp/%d/quic-v1", listenIP, cfg.Port),
```

Comment:
does this actually enable both transports? I would expect you'd need to use `libp2p.Transport` https://pkg.go.dev/github.com/libp2p/go-libp2p#Transport

Reply:
Yes. The default transports include websockets, tcp, and 2 versions of QUIC, so it might be more correct to say that this filters out the transports which we don't use.